test/functional/web/safari-nativewebtap-e2e-specs.js

```diff
@@ -1,270 +1,268 @@
+
+            await driver.$(`=${PAGE_3_LINK}`).click();
+
+            await spinTitleEquals(driver, PAGE_3_TITLE, SPIN_RETRIES);
+
+            await driver.back();
+
+            // try again, just to make sure
+            await driver.$(`=${PAGE_3_LINK}`).click();
 
-//             await spinTitleEquals(driver, PAGE_3_TITLE, SPIN_RETRIES);
-//           });
+            await spinTitleEquals(driver, PAGE_3_TITLE, SPIN_RETRIES);
+          });
 
-//           it('should be able to bypass measuring the offset', async function () {
-//             await driver.updateSettings({
-//               nativeWebTapStrict: true,
-//             });
-
-//             const start1 = performance.now();
-//             await loadPage(driver, GUINEA_PIG_PAGE);
-//             await driver.elementByLinkText(PAGE_3_LINK).click();
-//             const end1 = performance.now();
-//             const durationWithoutControlled = end1 - start1;
-
-//             await driver.updateSettings({ nativeWebTapTabBarVisibility: 'visible' });
-//             const start2 = performance.now();
-//             await loadPage(driver, GUINEA_PIG_PAGE);
-//             await driver.elementByLinkText(PAGE_3_LINK).click();
-//             const end2 = performance.now();
-//             const durationWithControlled = end2 - start2;
-//             durationWithControlled.should.be.below(durationWithoutControlled);
-//           });
-
-//           it('should be able to tap on an element after scrolling', async function () {
-//             await loadPage(driver, GUINEA_PIG_SCROLLABLE_PAGE);
-//             await driver.execute('mobile: scroll', {direction: 'down'});
-
-//             await driver.elementByLinkText(PAGE_3_LINK).click();
-
-//             await spinTitleEquals(driver, PAGE_3_TITLE, SPIN_RETRIES);
-//           });
-
-//           it('should be able to tap on an element after scrolling, when the url bar is present', async function () {
-//             await loadPage(driver, GUINEA_PIG_SCROLLABLE_PAGE);
-//             await driver.execute('mobile: scroll', {direction: 'down'});
+          it('should be able to bypass measuring the offset', async function () {
+            await driver.updateSettings({
+              nativeWebTapStrict: true,
+            });
+
+            const start1 = performance.now();
+            await loadPage(driver, GUINEA_PIG_PAGE);
+            await driver.$(`=${PAGE_3_LINK}`).click();
+            const end1 = performance.now();
+            const durationWithoutControlled = end1 - start1;
+
+            await driver.updateSettings({ nativeWebTapTabBarVisibility: 'visible' });
+            const start2 = performance.now();
+            await loadPage(driver, GUINEA_PIG_PAGE);
+            await driver.$(`=${PAGE_3_LINK}`).click();
+            const end2 = performance.now();
+            const durationWithControlled = end2 - start2;
+            durationWithControlled.should.be.below(durationWithoutControlled);
```

Comment:
I'm not sure if we should keep this assertion because I sometimes saw an error as

```
  2) iPad simulator -
       with tabs -
         should be able to bypass measuring the offset:

      AssertionError: expected 2752.6892499923706 to be below 2676.3116250038147
      + expected - actual

      -2752.6892499923706
      +2676.3116250038147
      
      at Context.<anonymous> (test/functional/web/safari-nativewebtap-e2e-specs.js:220:46)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```